metecho/api/serializers.py

```diff
@@ -117,6 +119,44 @@ class Meta:
         fields = ("id", "username", "avatar_url")
 
 
+class GitHubOrganizationSerializer(serializers.ModelSerializer):
+    id = serializers.CharField(read_only=True)
+
+    class Meta:
+        model = GitHubOrganization
+        fields = ("id", "name")
+
+
+class ProjectCreateSerializer(serializers.ModelSerializer):
+    organization = serializers.PrimaryKeyRelatedField(
+        queryset=GitHubOrganization.objects.all(), pk_field=serializers.CharField()
+    )
+
+    class Meta:
+        model = Project
+        fields = ("name", "description", "organization", "repo_name", "github_users")
+
+    def validate_organization(self, organization):
+        gh_user = gh_given_user(self.context["request"].user)
+        gh_user_orgs = [org.login for org in gh_user.organizations()]
+        if organization.login not in gh_user_orgs:
+            raise serializers.ValidationError(
+                _(
+                    "Either you are not a member of the %(name)s organization "
+                    "or it hasn't installed the Metecho GitHub app"
+                )
+                % {"name": organization}
+            )
+        return organization
```

Comment:
@davisagli this validation should only allow users that are members of the selected `organization` to create new projects/repos on it. There's a little ambiguity in the error message because an org might be missing even when the user is actually a member of it.

Reply:
I ended up moving this logic to a separate job so the frontend can check the user's membership as soon as they select an org from the dropdown (see aeec090678d122b83d5ab77f5c77fa61720dbc63)